drives.go

```diff
@@ -103,3 +103,15 @@ func WithRateLimiter(limiter models.RateLimiter) DriveOpt {
 		d.RateLimiter = &limiter
 	}
 }
+
+// WithCacheType sets the cache strategy for the block device
+func WithCacheType(cache_type string) DriveOpt {
```

Comment:
Generally variables in Go are camelCase.

Reply:
Ack. Will change to camelCase